HicServices/RDMP
# Task/rdmp 122 update a database extraction

* Adds ability to re-extract a project to a database
* Extract to database now respects the "IsExtractionPrimaryKey" flag on column items

How To Test:
* Create a simple catalogue and extract it to a database
* Update the catalogue with new data and attempt to re-extract 
* The new data should be extracted into the target database
